app/controllers/market_models_controller.rb

```diff
@@ -56,6 +57,11 @@ def destroy
 
   private
 
+  def clear_business_case_jobs
```

Comment:
This would be a good candidate for a service object, e.g.:
```ruby
if MarketModel::Update.run(market_model, market_model_params)
  redirect_to ... # success
else
  render :edit # validation errors
end
```

(like [TestingGround::SaveAs](https://github.com/quintel/etmoses/blob/4c45e7383e74d9d8151384981ab04fa8576003f9/app/services/testing_ground/save_as.rb)).